timon/plugins/http_server/views.py

```diff
@@ -24,6 +24,8 @@
 
 app = Quart(__name__)
 
+BASE_URL_ROUTE = "/api"
```

Comment:
hmmm I would have said that the base url is "/" or empty string by default.

and then you have /api for all api urls
and if you ever add static links they would be at /static

and if somebody wants to install behind a reverse proxy theynext to another web app they could decide to habe a baseurl of
"/timon"  which results in "/timon/api" and "/timon/static"